### PR TITLE
Corrects missing slash in a bullet

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request

Were the bullets even working?

## Why It's Good For The Game

Fixes a missing slash

## Changelog
:cl:
code: Corrects the missing slash in 10mm sleeping bullets, also removes double vars that are not needed
/:cl:

### CODE REVIEW

File: modular_citadel/code/modules/projectiles/guns/ballistic/handguns.dm [hunk 1]
```diff
@@ -58,12 +58,9 @@
 
 //////10mm soporific bullets//////
 
-obj/item/projectile/bullet/c10mm/soporific
+/obj/item/projectile/bullet/c10mm/soporific
 	name ="10mm soporific bullet"
-	armour_penetration = 0
```
Comment (Putnam3145) [inline modular_citadel/code/modules/projectiles/guns/ballistic/handguns.dm:63]: these are definitely doing something

**Review by Putnam3145 — CHANGES_REQUESTED** [review]

File: modular_citadel/code/modules/projectiles/guns/ballistic/handguns.dm (hunk 1 above)
Comment (Trilbyspaceclone) [reply modular_citadel/code/modules/projectiles/guns/ballistic/handguns.dm:63]: Nope they have 0 ap its a subtupe of 10mm that has none thats a subtype of bullet that has none

Comment (necromanceranne) [reply modular_citadel/code/modules/projectiles/guns/ballistic/handguns.dm:63]: Yeah fairly sure this was unnecessary and was being inherited and removal cuts down on fat. I'd double check but yeah 10mm already had 0.

**Review by Putnam3145 — APPROVED** [review]
Alright, further review shows on the level

**Pull request merged**